lib/modulr/api/customers_service.rb

```diff
@@ -1,12 +1,40 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
+
 module Modulr
   module API
     class CustomersService < Service
       def find(id:)
         response = client.get("/customers/#{id}")
         Resources::Customers::Customer.new(response.env[:raw_body], response.body)
       end
+
+      def create(type:, legal_entity:, **opts)
+        payload = {
+          type: type,
+          legalEntity: legal_entity,
+        }
+
+        payload[:externalReference] = opts[:external_reference] if opts[:external_reference]
+        payload[:name] = opts[:name] if opts[:name]
+        payload[:companyRegNumber] = opts[:company_reg_number] if opts[:company_reg_number]
+        payload[:registeredAddress] = opts[:registered_address] if opts[:registered_address]
+        payload[:tradingAddress] = opts[:trading_address] if opts[:trading_address]
+        payload[:industryCode] = opts[:industry_code] if opts[:industry_code]
+        payload[:tcsVersion] = opts[:tcs_version] if opts[:tcs_version]
+        payload[:expectedMonthlySpend] = opts[:expected_monthly_spend] if opts[:expected_monthly_spend]
+        payload[:associates] = opts[:associates] if opts[:associates]
+        payload[:documentInfo] = opts[:document_info] if opts[:document_info]
+        payload[:provisionalCustomerId] = opts[:provisional_customer_id] if opts[:provisional_customer_id]
+        payload[:customerTrust] = opts[:customer_trust] if opts[:customer_trust]
+        payload[:taxProfile] = opts[:tax_profile] if opts[:tax_profile]
+
+        response = client.post("/customers", payload)
+        Resources::Customers::Customer.new(response.env[:raw_body], response.body)
+      end
```

Comment:
```suggestion
      def create(**opts)
        response = client.post("/customers", opts)
        Resources::Customers::Customer.new(response.env[:raw_body], response.body)
      end
```

Reply:
I totally agree with that, but the thing is, in `lib/modulr/api/accounts_service.rb` (code below) we are already doing that kind of renaming, so for consistency I followed the same pattern...

```rb
def create(customer_id:, currency:, product_code:, **opts)
    payload = {
        currency: currency,
        productCode: product_code,
    }

    payload[:externalReference] = opts[:external_reference] if opts[:external_reference]

    response = client.post("/customers/#{customer_id}/accounts", payload)
    Resources::Accounts::Account.new(response.env[:raw_body], response.body)
end
```